www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -37,7 +37,8 @@
  *
  * @param className string : tag class name
  * @param altFormat string : format of the alternative field
- * @param defaultDate string : datepicker parameter of the setDate
+ * @param defaultDate string : GMT YYYY-MM-DDTHH:mm:ss timestamp
+ * @todo following 2 parameters seem to never be used, to remove ?
```

Comment:
these 2 parameters are required to be able to convert the timestamp from the slug in MBI module

Reply:
Thank you @sc979 for this, I'm not used to MBI, so I was not sure.
I then just removed these 2 `@todo` comments.